patterns/dswp-icon-with-excerpt.php

```diff
@@ -12,21 +12,18 @@
     <div class="wp-block-columns" style="padding-right:0;padding-left:0"><!-- wp:column -->
         <div class="wp-block-column"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical"}} -->
             <div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-                <div class="wp-block-group" style="padding-top:0;padding-bottom:0"><!-- wp:group {"className":"dswp-excerpt-img-group","layout":{"type":"constrained"}} -->
-                    <div class="wp-block-group dswp-excerpt-img-group"><!-- wp:image {"id":15469,"sizeSlug":"full","linkDestination":"none","className":"dswp-excerpt-img size-full is-resized is-style-default"} -->
-                        <figure class="wp-block-image size-full dswp-excerpt-img is-resized is-style-default"><img src="<?php echo esc_url( get_template_directory_uri() . '/assets/images/square-256.png' ); ?>" alt="" class="wp-image-15469" /></figure>
-                        <!-- /wp:image -->
-                    </div>
-                    <!-- /wp:group -->
+                <div class="wp-block-group" style="padding-top:0;padding-bottom:0"><!-- wp:image {"id":15469,"width":"54px","height":"auto","sizeSlug":"full","linkDestination":"none","align":"center","className":"dswp-excerpt-img size-full is-resized is-style-default"} -->
```

Comment:
There are still blocks using the `dswp-excerpt-img` class but it looks like that class was removed from the CSS.